tools/explorer/app.py

```diff
@@ -11,13 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Main module for DBO explorer."""
+import ast
+import pyfiglet
 import sys
+from termcolor import colored
+from typing import List
 
-from lib import arg_parser
 from lib import explorer
 from lib import model
-import pyfiglet
+from lib.model import StandardField
+from lib import arg_parser
+
+def _InputFields(ontology) -> List[StandardField]:
+  standard_field_list = []
+  field_number = input('how many fields do you want to input?\n')
```

Comment:
match?

Reply:
I'm imorting model which contains match. Should I specifically import match?